.github/workflows/deploy-graphql-gateway.yml

```
@@ -0,0 +1,308 @@
+          aws-region: ${{ env.AWS_REGION }}
+          audience: sts.amazonaws.com
+          role-session-name: gha-health-check-${{ github.run_id }}
+
+      - name: Update kubeconfig
+        run: |
+          aws eks update-kubeconfig \
+            --region ${{ env.AWS_REGION }} \
+            --name nova-staging
+
+      - name: Health check
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:6:30: Double quote to prevent globbing and word splitting

(shellcheck)